src/main/kotlin/com/demonwav/mcdev/util/reference/ReferenceResolver.kt

```diff
@@ -95,8 +95,8 @@ fun LookupElementBuilder.completeToLiteral(context: PsiElement): LookupElementBu
     // TODO: Currently we replace everything with a single PsiLiteral,
     // not sure how you would keep line breaks after completion
     return withInsertHandler({ context, item ->
-                context.laterRunnable = ReplaceElementWithLiteral(context.editor, context.file, item.lookupString)
-            })
+        context.laterRunnable = ReplaceElementWithLiteral(context.editor, context.file, item.lookupString)
+    })
```

Comment:
Prefer the `{}` syntax over `({})` where it makes sense and where applicable. This is a spot  where I'd say it makes sense.